src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -5620,6 +5620,30 @@ Class C
             Await VerifyItemInLinkedFilesAsync(markup, "x", expectedDescription)
         End Function
 
+        <WorkItem(13161, "https://github.com/dotnet/roslyn/issues/13161")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function CommitGenericOnTab() As Task
+            Dim text =
+<code>
+Class G(Of T)
+End Class
+
+Class DG
+    Function Bar() as $$
+End Class</code>.Value
+
+            Dim expected =
+<code>
+Class G(Of T)
+End Class
+
+Class DG
+    Function Bar() as G(Of
+End Class</code>.Value
+
+            Await VerifyProviderCommitAsync(text, "G(Of …)", expected, Nothing, "")
```

Comment:
That's the displaytext of the item to commit. The local `expected` contains the expected text which has no `...`.